lib/pbench/server/database/models/server_config.py

```diff
@@ -213,22 +213,22 @@ class ServerConfig(Database.Base):
     def _default(key: str) -> JSONVALUE:
         try:
             config = SERVER_CONFIGURATION_OPTIONS[key]
-        except KeyError:
+        except KeyError as e:
             if key:
-                raise ServerConfigBadKey(key)
+                raise ServerConfigBadKey(key) from e
             else:
-                raise ServerConfigMissingKey()
+                raise ServerConfigMissingKey() from e
```

Comment:
This is just itching to be a ternary:
```
        except KeyError as e:
            raise ServerConfigBadKey(key) if key else ServerConfigMissingKey() from e
```
...but I'm sure that Black will trash it.

Also, as long as I'm scratching itches here, it seems like the argument to `ServerConfigBadKey()` should be `e` (or maybe `str(e)`) rather than `key`...but `key` is, of course, fine.